osu.Game/Screens/Play/Player.cs

```diff
@@ -50,7 +50,13 @@ public class Player : ScreenWithBeatmapBackground
 
         public override bool HideOverlaysOnEnter => true;
 
-        public override OverlayActivation InitialOverlayActivationMode => OverlayActivation.UserTriggered;
+        private const OverlayActivation initial_overlay_activation_mode = OverlayActivation.UserTriggered;
+        public override OverlayActivation InitialOverlayActivationMode => initial_overlay_activation_mode;
+
+        /// <summary>
+        /// The current activation mode for overlays.
+        /// </summary>
+        protected readonly Bindable<OverlayActivation> OverlayActivationMode = new Bindable<OverlayActivation>(initial_overlay_activation_mode);
```

Comment:
exposing this is really confusing. i'd hope it was private.

Reply:
This bindable is marked as protected only for testing purposes, as we can't resolve an `OsuGame` instance through DI in tests. (see this https://github.com/ppy/osu/pull/9766#discussion_r464548375 for reference)